virtualmachine/vsphere/util.go

```diff
@@ -404,43 +404,118 @@ var findVM = func(vm *VM, dc *mo.Datacenter, name string) (*mo.VirtualMachine, e
 	return moVM, vm.answerQuestion(moVM)
 }
 
-func searchTree(vm *VM, mor types.ManagedObjectReference, name string) (*mo.VirtualMachine, error) {
-	switch mor.Type {
-	case "Folder":
+// splitPathToList: splits path containing special character including delimiter
+// for slash "/" (\/) and return slice of strings containing folder/vm names
+// for eg:  "vms\/test\/rec/rec\/1/rhel\/template\/vm"	: [vms/test/rec rec/1 rhel/template/vm]
+// Directory structure: vm/test/rec -> rec/1 -> rhel/template/vm
+func splitPathToList(path string) []string {
+	pathList := make([]string, 0)
+
+	// split at escaped '/', if none length of returned slice will be 1
```

Comment:
"none"? typo?

Reply:
it means if no escaped '/' are present. will update the comment.